api_docs/swagger/paths/partner_orders/renewal.yaml

```diff
@@ -0,0 +1,28 @@
+post:
+  tags:
+    - partner_orders
+  parameters:
+    - $ref: '/swagger/swagger.yaml#/components/parameters/Date'
+    - $ref: '/swagger/swagger.yaml#/components/parameters/ContentMd5'
+    - $ref: '/swagger/swagger.yaml#/components/parameters/Authorization'
+  summary: Renew an OEM partner order
+  description: |-
+    The attribute `partner_order_items` should contain only items with SKUs
+    related to subscriptions created on the previous order (with the same OEM
+    token). It means the endpoint can not be used to create new subscriptions,
+    only to extend the lifespan of active subscriptions and/or change the system
+    limits.
+  requestBody:
+    description: Partner order object
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '/swagger/swagger.yaml#/components/requestBodies/PartnerOrderPayload'
+  responses:
+    '200':
+      description: Partner order created successfully.
+      content:
+        application/json:
+          schema:
+            $ref: '/swagger/swagger.yaml#/components/schemas/PartnerOrder'
```

Comment:
From the call today, did I understand correctly that renewal partner orders do not need to receive a `starts_at` nor a the end user account number?

Reply:
Yes, this is right. And also I understand that there'll be no use for the `expires_at` date once subscriptions' lifespans will be computed using the SKU duration.

Reply:
Ok; I _think_ we are still displaying those fields in the docs for the renewal endpoint though, aren't we?

Reply:
Yes, the `PartnerOrder` schema is the same in all endpoints, just the payloads differ.

Reply:
Oh, yes, I commented on the wrong line, sorry! I meant about the payload. Are the different payloads for new/renew orders documented?

Reply:
Yes, if you check the request body section of each operation, there is `Example value`, and then you can switch to `Schema`

![image](https://user-images.githubusercontent.com/8546650/114209172-b5a87f80-9934-11eb-83d7-cb5cb457c0af.png)

Reply:
I'm improving the field and operations descriptions to match the the same style used in the SCC API Swagger.